Classes/Demo/HomeDataController.swift

```diff
@@ -70,11 +72,13 @@ enum HomeAccountItemIdentifier: Hashable {
 enum HomeDataControllerEvent {
     case didUpdate(HomeDataController.Updates)
     case deliverASARequestsContentUpdate(IncomingASAsRequestList?)
+    case didUpdateSpotBanner(String?)
     
     var snapshot: HomeDataController.Snapshot {
         switch self {
         case .didUpdate(let updates): return updates.snapshot
-        case .deliverASARequestsContentUpdate(_): return .init()
+        case .deliverASARequestsContentUpdate: return .init()
+        case .didUpdateSpotBanner: return .init()
```

Comment:
Using typed initialisers usually increase readability. Consider using `ElementType()` instead of `.init()`.